modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -42,9 +42,56 @@ To successfully install ROSA clusters, use latest version of the ROSA CLI.
 +
 [source,terminal]
 ----
-$ rosa create account-roles --mode manual <1>
+$ rosa create account-roles --interactive --mode manual <1><2>
```

Comment:
Not critical, but if you need to add a second callout, you could consider breaking this up on two lines so that you have a callout on each line (and next to the applicable item)

Reply:
Resolved.